team-telnyx/webrtc
# feat(WEBRTC-478):  hide telnyxIDs and clientState on public docs

- hide telnyxIDs and clientState on public docs, these features are in WIP.

## 📝 To Do


## ✋ Manual testing

1. Navigate into `packages/js`
2. Run `npm i && npm run docs`
3. See the generate docs on `js/docs`
4. The clientState and telnyxIDs should not be included on MD files. 

## 🦊 Browser testing

### Desktop


## 📸 Screenshots

| Description | Screenshot |
| ----------- | ---------- |
| Desktop     |            |
| usage.gif   |            |
